packages/react-dom-18/rollup.config.js

```diff
@@ -29,64 +35,37 @@ const pluginsAlias = [
   nodeResolve(),
   commonjs({ include: /node_modules/ }),
   replace({ preventAssignment: true, values: { "process.env.NODE_ENV": JSON.stringify(env) } }),
-  env === "production" ? terser({ format: { comments: false } }) : null,
-];
-
-const pluginsSpecificVersion = [
-  eikPlugin({
-    maps: [
-      {
-        imports: {
-          react: `https://assets.finn.no/npm/${name}/${version}/react.${env}.min.js`,
+  env === "production"
+    ? swc({
+        swc: {
+          env: {
+            targets: browserslist,
+          },
         },
-      },
-    ],
-  }),
-  nodeResolve(),
-  commonjs({ include: /node_modules/ }),
-  replace({ preventAssignment: true, values: { "process.env.NODE_ENV": JSON.stringify(env) } }),
+      })
+    : null,
   env === "production" ? terser({ format: { comments: false } }) : null,
 ];
 
 export default [
   {
     input: "./react-dom.js",
     output: { format: "esm", sourcemap: true, file: `./dist/react-dom.${env}.js` },
-    plugins: pluginsAlias,
+    plugins,
   },
   {
     input: "./react-dom.client.js",
     output: { format: "esm", sourcemap: true, file: `./dist/react-dom.client.${env}.js` },
-    plugins: pluginsAlias,
+    plugins,
   },
   {
     input: "./react-dom.server.browser.js",
     output: { format: "esm", sourcemap: true, file: `./dist/react-dom.server.browser.${env}.js` },
-    plugins: pluginsAlias,
+    plugins,
   },
   {
     input: "./react-dom.server.node.js",
     output: { format: "esm", sourcemap: true, file: `./dist/react-dom.server.node.${env}.js` },
-    plugins: pluginsAlias,
-  },
-  {
-    input: "./react-dom.js",
-    output: { format: "esm", sourcemap: true, file: `./dist/react-dom.${version}.${env}.js` },
-    plugins: pluginsSpecificVersion,
-  },
-  {
-    input: "./react-dom.client.js",
-    output: { format: "esm", sourcemap: true, file: `./dist/react-dom.client.${version}.${env}.js` },
-    plugins: pluginsSpecificVersion,
-  },
-  {
-    input: "./react-dom.server.browser.js",
-    output: { format: "esm", sourcemap: true, file: `./dist/react-dom.server.browser.${version}.${env}.js` },
-    plugins: pluginsSpecificVersion,
-  },
-  {
-    input: "./react-dom.server.node.js",
-    output: { format: "esm", sourcemap: true, file: `./dist/react-dom.server.node.${version}.${env}.js` },
-    plugins: pluginsSpecificVersion,
+    plugins,
```

Comment:
Matches the current file structure on Eik:
https://assets.finn.no/npm/react-dom/18.2.0